client/my-sites/comments/comment-list/index.jsx

```diff
@@ -157,6 +161,12 @@ export class CommentList extends Component {
 		const { commentId, postId } = comment;
 		const { isUndo, doPersist, showNotice } = options;
 
+		if ( isUndo ) {
+			this.setState( { lastUndo: commentId } );
```

Comment:
Good point. I started with a list, then dropped it because it was overkill, but I didn't think about bulk!

For the sake of conciseness, I'll only add a TODO comment for now.